src/main/java/com/develhope/spring/services/UniversalFieldUpdater.java

```diff
@@ -0,0 +1,42 @@
+package com.develhope.spring.services;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class UniversalFieldUpdater {
```

Comment:
A cosa serve questo ???

Reply:
Qui posso metterr qualsiasi classe come source ad esempio dto, e target ad esempio oggetto normale, si può utilizzare per update ( controlla tutti get, se sono validi aggiorna questo campo in oggetto target, così posso aggiornare o solo alcuni campi o tutti)

Reply:
Come mai no ModelMapper ?

Reply:
> A cosa serve questo ??? 

Semplicemente chiama getters controlla, imposta setter con lo stesso nome e tipo di ritorno. Se ci sono campi che non presentano gli salta e poi se c'è bisogno devo impostare manualmente,
È altra cosa, che non lo ancora applicato in tutti casi, dove si può. Ma lo farò 🙂 forse sono pazzo e esiste il metodo più facile, ma io vorrei fare le prove di qualcosa nuovo

Reply:
Model mapper non fa i controlli su ogni campo

Reply:
Ti rispondo su Discord